compose/compose-file/index.md

```diff
@@ -882,6 +882,73 @@ port (a random host port will be chosen).
      - "127.0.0.1:5000-5010:5000-5010"
      - "6060:6060/udp"
 
+### secrets
+
+After you
+[define or reference secrets in the top-level `secrets` configuration](#secrets-configuration-reference),
+grant access to secrets on a per-service basis using the per-service `secrets`
+configuration. Two different syntax variants are supported.
+
+#### Short syntax
+
+The short syntax variant only specifies the secret name. This grants the
+container access to the secret and mounts it at `/run/secrets/<secret_name>`
+within the container. The source name and destination mountpoint are both set
+to the secret name.
+
+> **Warning**: Due to a bug, using the short syntax currently mounts the secret
+> with permissions `000`, which means the secret is currently unreadable if you
+> define it using the short syntax. The workaround is to use the long syntax
+> instead.
+
+The following example uses the short syntax to grant the `redis` service
+access to the `my_secrets` secret, which has been defined as an external
```

Comment:
Nitpick: For consistency with `volumes` and `networks`, the examples should probably use local secrets over external ones (which require additional knowledge of the Docker Engine/CLI to use)